DFE-Digital/early-careers-framework
# [CPDLP-859] NPQ current/payable statements

## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDLP-859

- Adds current and payable views to NPQ payment breakdowns
- `Invoice` has been renamed to `Statement` to related more closely to business terms
- For a statement payment breakdown for a course is now nested within a statement. This nesting structure provides better navigation and will provide a more rigid structure when new statements will be added in the future.
- May also coincidentally fix https://github.com/DFE-Digital/early-careers-framework/pull/1628

## Screenshots

![Screenshot 2022-01-17 at 16 30 22](https://user-images.githubusercontent.com/92580/149807076-08109a3a-e06f-409e-ba85-1f36bfd5259f.png)

## Tech review

### Is there anything that the code reviewer should know?

### Code quality checks

### HTML Checks

### Gotchas

## Product review

### How can someone see it it review app?

- https://ecf-review-pr-1634.london.cloudapps.digital/finance
- Login as a finance user
- View NPQ payment breakdowns
- Should be able to toggle between current and payable views

## External API changes

- None
